lib/Runtime/Library/JavascriptDate.cpp

```diff
@@ -1503,16 +1503,6 @@ namespace Js
             DateImplementation::DateTimeFlag::NoDate);
     }
 
-    // CONSIDER: ToGMTString and ToUTCString is the same, but currently the profiler use the entry point address to identify
-    // the entry point. So we will have to make the function different. Consider using FunctionInfo to identify the function
```

Comment:
I'll set up a test to investigate. If not, how can we make it show the data separately while using the same function object for the two functions?